sdks/java/io/kafka/pom.xml

```diff
@@ -106,7 +106,7 @@
     <dependency>
       <groupId>org.apache.kafka</groupId>
       <artifactId>kafka-clients</artifactId>
-      <version>[0.9.0.1,0.11.0)</version>
```

Comment:
I agree with @davorbonaci's point about fixed version. Can we make this a mvn property so that we can easily test newer version with something like 'mvn test -Dkafka.clients.version=0.10.1'